src/main/webapp/view/video/video.js

```diff
@@ -16,40 +16,43 @@ import {startVideoSyncTimer} from '../../synchronizer.js';
 
 const SCRIPT = 'script';
 
-window.onYouTubeIframeAPIReady = onYouTubeIframeAPIReady;
+/** Initializes and stores video player information. */
+// TODO: Add function that returns current video time #259.
+export default class Video {
+  /** Loads YouTube iFrame API. */
+  async loadVideoApi() {
+    window.onYouTubeIframeAPIReady = this.onYouTubeIframeAPIReady.bind(this);
+    const videoApiScript = document.createElement(SCRIPT);
+    const firstScriptTag = document.getElementsByTagName(SCRIPT)[0];
+    videoApiScript.src = 'https://www.youtube.com/iframe_api';
+    firstScriptTag.parentNode.insertBefore(videoApiScript, firstScriptTag);
+  }
 
-/** Loads YouTube iFrame API. */
-export async function loadVideoApi() {
-  const videoApiScript = document.createElement(SCRIPT);
-  const firstScriptTag = document.getElementsByTagName(SCRIPT)[0];
-  videoApiScript.src = 'https://www.youtube.com/iframe_api';
-  firstScriptTag.parentNode.insertBefore(videoApiScript, firstScriptTag);
-}
-
-/**
- * Creates a YouTube Video iFrame that plays lecture video after
- * the API calls it. This is a required callback from the API.
- */
-// TODO: Change height and width.
-function onYouTubeIframeAPIReady() {
-  window.videoPlayer = new window.YT.Player('player', {
-    height: '390',
-    width: '640',
-    videoId: window.LECTURE.videoId,
-    events: {
-      onReady: onPlayerReady,
-    },
-  });
-}
+  /**
+   * Creates a YouTube Video iFrame that plays lecture video after
+   * the API calls it. This is a required callback from the API.
+   */
+  // TODO: Change height and width.
```

Comment:
nit: I would re-word this TODO to be more specific. 

Support dynamic video height and width.